execution_workflows/diffUTR/diffUTR_nextflow/results/pipeline_info/execution_timeline.html

```diff
@@ -0,0 +1,200 @@
+<!doctype html>
```

Comment:
It is great that the `results` are shown to us and that the method works, but I don't think this should be in the repository. 
Pplease remove the `results` directory. We should be able to run `test_data` on this method, once it is merged.